### PR TITLE
Add pss toggle back and upgrade prometheus-operator apps

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

* upgrades prometheus operator apps
* adds back pss toggle for capi WCs

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/observability-bundle/templates/observability-bundle-psp-override.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  values: |
+    global:
+      rbac:
+        pspEnabled: {{ not .Values.global.podSecurityStandards.enforced }}
```
Comment (QuantumEnigmaa) [inline helm/observability-bundle/templates/observability-bundle-psp-override.yaml:6]:
```suggestion
        pspEnabled: {{ if not .Values.global.podSecurityStandards.enforced }}
```

Is `not` alone enough ?

Comment (QuentinBisson) [reply helm/observability-bundle/templates/observability-bundle-psp-override.yaml:6]: Yes it is :)

Comment (QuentinBisson) [reply helm/observability-bundle/templates/observability-bundle-psp-override.yaml:6]: That actually negates the value

Comment (QuentinBisson) [reply helm/observability-bundle/templates/observability-bundle-psp-override.yaml:6]: See rendered CM:

```yaml
# Source: observability-bundle/templates/observability-bundle-psp-override.yaml
apiVersion: v1
data:
  values: |
    global:
      rbac:
        pspEnabled: true
    kube-prometheus-stack:
      grafana:
        rbac:
          pspEnabled: true
      kube-state-metrics:
        podSecurityPolicy:
          enabled: true
      prometheus-node-exporter:
        rbac:
          pspEnabled: true
    prometheus-agent:
      psp:
        enabled: true
    promtail:
      rbac:
        pspEnabled: true
    # For grafana-agent
    psp:
      enabled: true
kind: ConfigMap
metadata:
  name: "-observability-bundle-psp-override"
  namespace: "default"
```

Comment (QuantumEnigmaa) [reply helm/observability-bundle/templates/observability-bundle-psp-override.yaml:6]: Oh ok :)

**Review by QuantumEnigmaa — APPROVED** [review]

**Pull request merged**